api/helpers/gamestate/publish-game-state.js

```diff
@@ -40,6 +38,8 @@ module.exports = {
         return numPasses;
       };
 
+      const victory = await sails.helpers.gamestate.checkGameStateForWin(game, gameState, countPasses());
```

Comment:
I'm not a huge fan of passing the function so that it will call twice.

I think the move here is to immediatley invoke countPasses, and then pass it to victory, and the socket call below as just `countPasses`

```suggestion
const countPasses = (function () {
  let numPasses = 0;
  for (const gameState of game.gameStates.slice(-3)) {
    if (gameState.moveType !== MoveType.PASS) {
      return numPasses;
    }
    numPasses++;
  }
  return numPasses;
})();
```

Reply:
I'm up for either this or to declare the function in one line and then call it and capture the result in a separate variable afterwards before passing that to the checkWinGame function and using it below in the game object. 

Either way I'd like the variable to be named numPasses

Reply:
@haaarsh619 I'll leave it up to you whether to immediately invoke vs use two variables, but let's avoid calling the function twice